src/components/NotFound.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const NotFound = () => {
+    return <NoContent />;
```

Comment:
Also there is no need to write return if you do not have any processing inside of component
const NotFound = () => <NoContent />

will be fine

Reply:
Updated